client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -11,7 +11,10 @@ import isMainSiteOf from 'calypso/state/selectors/is-main-site-of';
  * @param  {number}    siteId    The ID of the site we're querying
  * @returns {boolean}             Whether site with id equal to siteId is a connected secondary network site
  */
-export default createSelector( ( state, siteId ) => {
-	const siteIds = Object.keys( getSitesItems( state ) );
-	return siteIds.some( ( mainSiteId ) => isMainSiteOf( state, mainSiteId, siteId ) );
-}, getSitesItems );
+export default createSelector(
+	( state, siteId ) => {
+		const siteIds = Object.keys( getSitesItems( state ) );
+		return siteIds.some( ( mainSiteId ) => isMainSiteOf( state, mainSiteId, siteId ) );
+	},
+	( state ) => [ getSitesItems( state ) ]
```

Comment:
Updating the dependency argument because TS throws the below error:

<img width="857" alt="Screenshot 2022-08-22 at 8 17 11 PM" src="https://user-images.githubusercontent.com/10586875/185950736-14a7dc68-806e-4fc1-af92-4a496aeecac4.png">